dockerscripts/docker-entrypoint.sh

```diff
@@ -41,12 +49,15 @@ docker_secrets_env() {
 
 ## Set KMS_MASTER_KEY from docker secrets if provided
 docker_kms_encryption_env() {
-    KMS_MASTER_KEY_FILE="/run/secrets/$MINIO_KMS_MASTER_KEY_FILE"
+    if [ -f "$MINIO_KMS_MASTER_KEY_FILE" ]; then
```

Comment:
`SSE_MASTER_KEY_FILE` also can also support custom paths

Reply:
It's deprecated whoever wants this newer feature use the newer ENV